### PR TITLE
[workers][SIMS #1397]Workers deployment

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Create CI/CD for workers;
- Added Zeebe env variables to the `sim-api-secrets` (it will be renamed later to `sims-secrets`);
- Added workers to docker-compose.
- Change local node version for docker from `node:16.13.0` to `node:16.17.0-alpine3.16`;
- Created a sample controller for ZeebeWorker methods;
- Created a sample service to get assessments from DB (to be removed);
- Created the `ZeebeTransportStrategy` to connect to the Camunda cloud and create the worker methods.

### CODE REVIEW

File: sources/packages/backend/apps/workers/src/controllers/assessment/assessment.dto.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface AssessmentDataWorkerInDTO {
```
Comment (ann-aot) [inline sources/packages/backend/apps/workers/src/controllers/assessment/assessment.dto.ts:1]: :)

**Review by ann-aot — APPROVED** [review]
👍

**Review by andrepestana-aot — APPROVED** [review]
LGTM!

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrewsignori-aot

**Pull request merged**